sopel/tools/__init__.py

```diff
@@ -55,21 +55,6 @@ def get_input(prompt):
         return raw_input(prompt).decode('utf8')
 
 
-def get_raising_file_and_line(tb=None):
```

Comment:
Why is this moved to the end of the file?

Reply:
Because it's deprecated and I wanted everything deprecated at the bottom of the file, together. I can totally revert that.

Reply:
It's mostly unnecessary diff noise, but it also affects blame output. Neither of which is a major concern for something deprecated, but still. I don't think moving deprecated code is a habit we should start. The decorator is enough to find it all. :)

Reply:
Fair enough, will revert then.

Reply:
Oh. Except that... I can't: the `deprecated` function is defined later in the file, so `get_raising_file_and_line` **must** be declared after.

Moving `deprecated` is even bigger. I'm not sure this is worth it either. :-/

Reply:
Well, then it's not "unnecessary diff noise". 😁

Python needs forward declarations. 😛